.travis.yml

```diff
@@ -25,24 +23,9 @@ jobs:
       node_js: lts/*
       script:
         - git checkout $TRAVIS_BRANCH
-        - npm run prettier
+        - npm run format:fix
         # commit changes and push back to branch on GitHub. If there are no changes then exit without error
         - 'git commit -a -m "style: prettier" && git push "https://${GH_TOKEN}@github.com/$TRAVIS_REPO_SLUG" ${TRAVIS_BRANCH} || true'
```

Comment:
We can move that to an action, too. I wrote about it here: https://dev.to/gr2m/automate-updates-of-prettier-standard-and-other-javascript-linting-tools-using-github-actions-nko

Reply:
Sure, we can. Although I'd prefer to do it in a separate PR instead of making this one even larger. Is it ok with you? Seems like it doesn't have to be merged at the same time.

Reply:
Absolutely, I'd suggest the same. Sorry, I should have said so